pilot/pkg/proxy/envoy/v2/config.go

```diff
@@ -41,19 +43,24 @@ import (
 )
 
 const (
-	// filterNameRouter is the name for the router filter.
-	filterNameRouter = "router"
+	// names for filters taken from envoy v2 API.
+	filterNameRouter            = "envoy.router"
+	filterNameCors              = "envoy.cors"
+	filterHTTPConnectionManager = "envoy.http_connection_manager"
+
+	ldsType = "type.googleapis.com/envoy.api.v2.Listener"
```

Comment:
use `cache.ListenerType`

Reply:
Same - let's not mix the implementations.